Birthday-2024-Project/ScenePrefabs/gallery.tscn

```diff
@@ -0,0 +1,400 @@
ded by cute girls, and has devoted Saplings and her handsome boy, Snail.", "The fabled golden apple is in the possession of the Keeper of Nature. One bite is enough to cure any ailment.", "“Come on, Sapling! It’s your favorite maid kirin with your favorite meal, a mac-n-cheese milkshake. Open up!”", "Oh, the bliss and comfort of Nemu onesie! These ears can put EleFauna’s ears to shame, and cuteness can overwhelm even the coldest hearts.", "Beloved kirin in her casual outfit, ready to stream :3 Is it time for sweaty gaming or ASMR? Who knows? Only time will tell.", "Faunashi-sensei in her golden pot and with her irreplaceable hammer will lead Baeruto and Moomsuke to the greatest heights of Getting Over It. Together with Orochimaru and SakuRyS, she will provide commentary for the participants in the GoI Trials! for the participants of GOI Trials!", "Matcha latte is one of the favorites of Fauna. Skilled baristas are making those in a matter of minutes, and you can be sure to also get cute foam art with it, just like this cute sapling.", "Agent Fauna7 refuses to use guns as bananas work just as well. It’s not because of her aim. It’s a false rumor spread by syndicates that will soon be done for. Right?", "It finally happened! 4th fes was the first time Fauna donned her idol outfit and sang on the big stage for everyone!", "A trusty companion of Agent Fauna7. It can do everything as long as it’s a distraction or smacking the head of an unsuspecting victim.", "The first glimpse of 3D Fauna was not what we expected, but a wide head and big horns will forever be in our minds.", "These are not oversized oven mitts. These are oversized horn warmers, and if you don’t think so, get ready for kirin headbutt attack.", "Just like any cat, Snail is a liquid and can end up in strange poses. And you, knowing all this, are staring at him! Give him some privacy!", "Where did Fauna go? She could be anywhere. She could be in her closet. She could be inside your walls. She could be right behind you! Or she could be somewhere here in the bushes.", "Did you know that green and purple colors are a great combination and are very pleasant for the eyes? Here we have Exhibit A: BEEGsmol Faunya in a purple dress, accompanied by the green of a spilled matcha latte.", "Even more matcha? What will I do with this whole powder? One glass is not enough. On the other hand, is it possible to overdose on it?", "Cowkini? Check. Safety cap? Check. Matcha latte? Che… Aaaaa… Uuuuu… That was not a good idea. Oh well…", "If, by any chance, your enemies have medieval castles, just follow EleFauna’s advice. “Prrrt. Prrrt. Toot~ Elephants, roll out!” It works every time.", "“Okaerinasaimase, Goshujin-sama! Your... your... your maid is here. It was me all along. Aaaaaaa! What do you think? What do you think?”", "\"Konfauna~ Your gaming idol kirin Ceres Fauna is here!\" The Keeper of Nature, a druidic kirin who appeared on the internet to win over humans and convince them to return to nature."])
```

Comment:
"she will provide commentary for the participants in the GoI Trials! for the participants of GOI Trials!" remove one instance of "for the participants of GOI Trials!"

Reply:
I've updated in doc "is it possible to overdose on it?" to "is it possible to overdose it?". Should also be pulled here

Reply:
Fixed both instances of incorrect descriptions